web/deeamon128/html/navigation-menu/app.js

```diff
@@ -0,0 +1,7 @@
+const menu = document.querySelector('#mobile-menu');
+const menuLinks = document.querySelector ('.nav-menu');
```

Comment:
remove space before (

`const menuLinks = document.querySelector('.nav-menu');`

Reply:
thanks for your feedback :D